app/components/ui/atoms/PersonPod.js

```diff
@@ -42,6 +48,10 @@ const StyledPod = styled(Flex)`
   height: 120px;
 `
 
+const CollapsibleBox = styled(Box)`
+  min-width: 0;
```

Comment:
Is it worth having a comment here, since both of us found the necessity for this piece of CSS surprising?